components/ADempiere/Form/VAllocation/components/Payments.vue

```diff
@@ -0,0 +1,277 @@
+          <div slot="header" class="clearfix" style="text-align: center;">
+            <b> {{ 'Pagos' }} </b>
+          </div>
+          <el-table
+            :data="tableData"
+            border
+            style="min-height: 400px;width: 100%;"
+          >
+            <el-table-column
+              v-for="(header, key) in headers"
+              :key="key"
+              prop="id"
+              :width="header.width"
+              :label="header.label"
+            />
+          </el-table>
+        </el-card>
+      </div>
+      <div style="width: 50%;">
+        <el-card style="height: 100%;">
+          <div slot="header" class="clearfix" style="text-align: center;">
+            <b> {{ 'Facturas' }} </b>
+          </div>
+          <el-table
+            :data="tableData"
+            border
+            style="min-height: 400px;width: 100%;"
+          >
+            <el-table-column
+              v-for="(header, key) in headers"
+              :key="key"
+              prop="id"
+              :width="header.width"
+              :label="header.label"
+            />
+          </el-table>
+        </el-card>
+      </div>
+    </el-card>
+    <el-card class="box-card">
+      <!-- <div slot="header" class="clearfix" style="text-align: center;">
+        <b>
+          {{ 'Detalles de los Pago' }}
+        </b>
+      </div> -->
+      <div id="description-payment" class="description-payment" style="display: flex;">
+        <el-card
+          class="box-card"
+          style="width: 50%;float: left;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Cantidad de Registro:"
+                >
+                  <b>
+                    5
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Total Factura:"
+                >
+                  <b>
+                    8.506,23
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Convertido:"
+                >
+                  <b>
+                    530,63
+                  </b>
+                </el-form-item>
+              </el-col>
+            </el-form>
+          </div>
+        </el-card>
+        <el-card
+          class="box-card"
+          style="width: 50%;float: right;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Pagado:"
```

Comment:
@elsiosanchez please add translation.

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)